### PR TITLE
Updating from feedback

### PR DESCRIPTION
**Pull request by joelgdwave**
Please consider this PR to resolve the student questions from 1/2021. The discussion should now be consistent with the Gradescope assignment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,16 @@ Demonstrates what happens when you send a QUBO to the D-Wave QPU
...
 ## Overview
 
-This program takes a 3-variable QUBO function and demonstrates what goes on "behind the scenes" as it runs on the QPU.  We see:
+This program takes a 3-variable QUBO function and demonstrates what goes on 
+"behind the scenes" as it runs on the QPU.  We see:
 
 1. The original QUBO.
 2. The equivalent Ising form.
-3. An embedded Ising form onto the QPU architecture using four qubits.
+3. An Ising form embedded onto the QPU architecture using four qubits.
```
Comment (vgoliber) [inline README.md:11]: I think this should read "The Ising form..." rather than "An Ising form..." since we are using the specific one from step 2, yes?

**Review by vgoliber — CHANGES_REQUESTED** [review]
Since you're using an image for the QUBO equation (basic_qubo.png), can you format it so that there is no line break in the middle of the equation?

It would also be nice to crop the basic_problem.png so that "Basic Problem Using QUBO Process" is not included.

File: README.md (hunk 1 above)
Comment (joelgdwave) [reply README.md]: This is fixed - I pushed it. Thanks.

Comment (joelgdwave) [reply README.md]: @vgoliber, I cropped the other photo. In terms of the one with the line break, alas, it was formulated on a half slide, and I am having trouble with the PowerPoint. I didn't solve it yet. Let me know how critical it is.

**Review by vgoliber — APPROVED** [review]
I can make that last edit myself, thanks.

**Pull request merged**